### DISCUSSION TITLE
Support For Class::Tiny

### DISCUSSION THREAD

**Discussion by Tomasu** [Camelcade/Perl5-IDEA#2814] [Ideas]
I was wondering if it would be possible to get support for code using Class::Tiny? I think the most useful bit of support would just include handling the `use Class::Tiny qw/some fields here/;`  statement to provide the available methods (in this case: 'some', 'fields' and 'here'). Maybe also point `MyClass->new()` at a BUILD or BUILDARGS sub if available.

**Comment by hurricup** [comment]
Moved to the issues https://github.com/Camelcade/Perl5-IDEA/issues/2816
